package.json

```diff
@@ -51,6 +51,7 @@
     ]
   },
   "devDependencies": {
+    "@wagtail/stylelint-config-wagtail": "0.0.4",
```

Comment:
Two things here:

- This should be using a caret (`^0.0.4`) like other dependencies, we use a lockfile so there is no need to pin exact versions.
- This is a really minor thing but I would expect this to be versioned with a minor version number, `0.1.0`, `0.1.4`, or `0.4.0`, not with the patch number alone. This is because patches should only be used to indicate "patch"-type changes to existing minor or major releases, and 0.0.0 isn't a valid release number. SemVer guidance is to start at 0.1.0: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase. I don't think this has any incidence in practice though.